fling/component/group/base_group.py

```diff
@@ -84,28 +85,30 @@ def append(self, client: ClientTemplate) -> None:
         """
         self.clients.append(client)
 
-    def aggregate(self, train_round: int, aggr_parameter_args: dict = None) -> int:
+    def aggregate(self, train_round: int, participate_clients_ids: list, aggr_parameter_args: dict = None) -> int:
```

Comment:
Set the ``participate_clients_ids`` to be optional. If this argument is not provided, all clients are required to be updated.